packages/xo-server/src/api/host.js

```diff
@@ -224,8 +224,15 @@ export async function isHostServerTimeConsistent({ host }) {
   try {
     await this.getXapi(host).assertConsistentHostServerTime(host._xapiRef)
     return true
-  } catch (e) {
-    return false
+  } catch (err) {
+    if (
+      err.message.startsWith(
+        'host server time and XOA date are not consistent with each other'
+      )
```

Comment:
This test appears fragile, IMHO, it would be best to split `assertConsistentHostServerTime` into one method that throws and one that does not.